torch/_refs/nn/functional/__init__.py

```diff
@@ -485,3 +489,13 @@ def prelu(a: TensorLikeType, weight: TensorLikeType) -> TensorLikeType:
     )
 
     return refs.where(a > 0, a, a * weight)
+
+
+def relu6(a: TensorLikeType, inplace: bool = False) -> TensorLikeType:
+    """
+    Reference implementation of torch.nn.functional.prelu
+    """
+    if inplace:
+        raise NotImplementedError
+
+    return refs.nn.functional.hardtanh(a, 0, 6)
```

Comment:
Oh interesting, I just used hardtanh here because that was the implementation does. We would want to be consistent with that right?

Reply:
I'd suggest using `clamp` because primTorch is an opportunity to improve the quality of implementations, but some reference implementers prefer fidelity since fidelity is often faster to implement, so there's no hard rule.

Either way I'd add a brief comment here about which choice you made and why